devel/lab/Portfile

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        zaquestion lab 0.14.0 v
+categories          devel
+platforms           darwin
+license             The Unlicense
+maintainers         {cwrinn @cwrinn}
```

Comment:
You are not a MacPorts committer, so your email address can't be cwrinn@macports.org (which is implicit there), you'll have to give your actual one. I also suggest that you should probably make this openmaintainer unless there's a good reason not to.

Reply:
No problem, I misunderstood the documentation then. I'll remove it.